the-collab-lab/tcl-70-smart-shopping-list
# Navbar Design

## Description

This code adds styling to the navbar. It also moves the darkmode button in mobile view

## Related Issue

closes #80 

## Acceptance Criteria

## Acceptance Criteria
- [x] The color of the logo needs to change to match the rest of the elements in the Navbar ('#fdeecd')
- [x] The sign out button needs to be styled
- [x] the darkmode button doesn't fit well on mobile screens so it can be moved up to the corner of the header
- [x] Change the app name to the font "Newsreader"

## Type of Changes
|     | Type                       |
| --- | -------------------------- |
|  ✓    | :sparkles: Enhancement     |


## Updates

### Before

![image](https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/28488515/818039c6-1a87-4de5-9f58-2929ab902fe9)

### After
![image](https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/28488515/91a1f785-b19b-4ed9-8d49-5b61bd7d4993)

## Testing Steps / QA Criteria

* From your terminal, pull down this branch with `git pull origin km-design-navbar` and check that branch out with `git checkout km-design-navbar` 
* Run `npm start`
* In the browser, go to  `http://localhost:3000/` to view the changes to the header
